### PR TITLE
add licensingquerysource crd

### PR DESCRIPTION
**Pull request by adamdyszy**
do not merge yet, need to test it first

### CODE REVIEW

File: config/samples/operator_v1_ibmlicensingquerysource.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: operator.ibm.com/v1
+kind: IBMLicensingDefinition
```
Comment (kacperkrzyzak) [inline config/samples/operator_v1_ibmlicensingquerysource.yaml:2]: It should be query source example i believe

**Review by kacperkrzyzak — APPROVED** [review]

File: config/samples/operator_v1_ibmlicensingquerysource.yaml (hunk 1 above)
Comment (adamdyszy) [reply config/samples/operator_v1_ibmlicensingquerysource.yaml]: good catch fixed in next commit

**Review by kacperkrzyzak — APPROVED** [review]

**Pull request merged**